### ISSUE TITLE
Please, add a small section to the README on how to install and use this tool

### ISSUE THREAD

**Issue by nbro** [checkstyle/checkstyle#14838]
The most important thing is not in the README. I want to start using this tool, but I am spammed with all these badges, and there are no instructions on how to install and use this tool, but just links and confusing texts. Yes, I know there's the documentation, but why not also have some useful instructions directly on the README on how to do what I said?

**Comment by rnveach** [comment]
If we added a link, it would probably point to https://checkstyle.org/beginning_development.html

**Comment by nrmancuso** [comment]
>The most important thing is not in the README.

>  just links and confusing texts.

I agree with these statements, and I am actively working on a PR to improve this situation by adding a "quick start" section to the README: https://github.com/checkstyle/checkstyle/pull/14514. Thanks for raising this issue.

> I want to start using this tool, but I am spammed with all these badges,

Yes, I mostly agree with this statement; the badges are distracting, and not really helpful to most users. However, this seems to be a fairly common Github practice, and easily ignored.

**Comment by nrmancuso** [comment]
> For just usage better
to start with https://checkstyle.org/
For starting contribution better to look at GitHub

Most projects (especially projects targeted at developers) have usage information as the first element of their README, and I wholeheartedly agree with this practice. However, this is sort of a complicated topic for us. Most folks will use checkstyle via gradle or maven plugins, which we do not maintain. I am not sure how I would feel about having the typical copy/pastable module in our README for gradle and maven dependencies since we have no control over these projects.

**Comment by romani** [comment]
readme is updated to show how to run it https://github.com/checkstyle/checkstyle/commit/abe223331fe85985505520db1c425fa42735d753#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

all CLI parameters are shown with examples of use https://checkstyle.org/cmdline.html#Examples
